repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -180,6 +220,76 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
     }
   }
 
+
+  def cullResponse(url: String, response: MutableHttpServletResponse, devicePermissions: DeviceList): Unit = {
+
+    def getJsPathFromString(jsonPath: String): JsPath = {
+      val pathTokens: List[PathToken] = JSONPath.parser.compile(jsonPath).getOrElse({
+        throw new ResponseCullingException(s"Unable to parse JsonPath: $jsonPath")
+      })
```

Comment:
Having both parenthesis and curly braces is redundant. Either one will work on it's own, but I'd probably go with parenthesis since you don't really need a code block for one statement.